common/src/main/java/com/firebase/geofire/GeoFire.java

```diff
@@ -212,7 +215,7 @@ public void onComplete(DatabaseError databaseError, DatabaseReference databaseRe
                 }
             });
         } else {
-            keyRef.setValue(null);
+            keyRef.setValueAsync(null);
```

Comment:
It looks like, that the firebase developer changed the name of that function if you do not add a CompletionListener. Before you could call setValue without listener, now you have to call setValueAsync if you do not want to use a listener. Since there is no listener required, i changed it to setValueAsync.

Reply:
Forgot to mention. I assume this is the fix for #28, since this function has been changed! (if you check some posts on stackoverflow.com, you will see that they suggest you to use a CompletionListener, which would call the 'old' function...